etc/greenboot/green.d/00_greenboot_notification.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -euo pipefail
 
-echo "Health Check SUCCESS! Boot status is GREEN"
+echo "<5>Health Check SUCCESS! Boot status is GREEN"
```

Comment:
does prepending with `<level>` really make it print out a message at that level ?